next.config.js

```diff
@@ -33,6 +33,46 @@ const nextConfig = {
       },
     ];
   },
+  async redirects() {
+    const routes = [
+      "account",
+      "tx",
+      "block",
+      "code",
+      "contract",
+      "proposal",
+      "project",
```

Comment:
should this be public-project to match the old route

Reply:
It is not necessary to make this a public project in order to match the old route.